app/assets/stylesheets/mtl/extend/_buttons.scss

```diff
@@ -0,0 +1,8 @@
+.btn-icon {
```

Comment:
okay, but then how about we'll simply add a `.btn-slim` similar to how btn-compressed works, but using `rem`, so it also works with `.large` et all?

```css
.btn-slim {
  padding: 0 1rem
}
```

usage then as follows:

```html
<a href="#" class="btn btn-slim">
  <i class="material-icons">cloud</i>
</a>
```